src/smt/theory_str_noodler/aut_assignment.h

```diff
@@ -54,6 +54,16 @@ namespace smt::noodler {
             return nfa;
         }
 
+        Mata::Nfa::Nfa sigma_automaton() const {
```

Comment:
We should add this function into Mata in the future.

Reply:
Yes, agree; hopefully to the builder namespace 😄